src/algebra/big_operators/intervals.lean

```diff
@@ -38,6 +39,22 @@ lemma prod_Ico_add [ordered_cancel_add_comm_monoid α] [has_exists_add_of_le α]
   (∏ x in Ico a b, f (c + x)) = (∏ x in Ico (a + c) (b + c), f x) :=
 @sum_Ico_add _ (additive β) _ _ _ _ f a b c
 
+lemma sum_range_neg_one_pow [ring β] {n : ℕ} :
```

Comment:
Yes, I can switch to `neg_one_geom_sum`. I might open a PR changing the name to `geom_sum_neg_one` though. The current name is inconsistent with everything else in the file, and searching for "sum_neg_one" turns up nothing at the moment.

Also, would making the definition of `geom_sum` reducible allow others to find these kind of lemmas via `library_search`? It's usually the first thing I try, but it fails more often than I would expect because of cases like this.